app/models/admin_profile.rb

```diff
@@ -5,9 +5,9 @@ class AdminProfile < BaseProfile
 
   def self.create_admin(full_name, email, sign_in_url)
     user = User.new(full_name: full_name, email: email)
-    user.confirm
 
     ActiveRecord::Base.transaction do
+      user.confirm
```

Comment:
Just curious if this isn't actually calling save. Most likely it is.

Reply:
My other comment which I deleted was about checking this confirmed in the unit test. We missed that in the first pr but it's small enough to add here